packages/ui-design-system/src/Kbd/Kbd.tsx

```diff
@@ -0,0 +1,13 @@
+import clsx from 'clsx';
+
+export function Kbd({ className, ...props }: React.ComponentProps<'kbd'>) {
```

Comment:
what does Kbd stand for?

Reply:
This is based on the [HTML tag with the same name](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd). It's a semantic comp that should define keyboard shortcut a user can use to trigger some actions.